cmd/add_container.go

```diff
@@ -154,14 +158,12 @@ func AddContainer(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-
-	err = generatePipelineDirFiles(workflowDirectory, pipelineName, containerName)
-	if err != nil {
+	if err := generatePipelineDirFiles(dir, workflowPath, pipelineName, containerName); err != nil {
+		fmt.Println(dir, workflowPath, pipelineName, containerName)
```

Comment:
What does this println do for us? was this a debug line or are we hoping for a error message here?

Reply:
def a debug line, good catch!